core-web/libs/ui/src/lib/dot-field-required/dot-field-required.directive.ts

```diff
@@ -0,0 +1,29 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
```

Comment:
Please add doc, this directives is used for, what do the input.

Reply:
Noted

Reply:
Updated